governance/program/src/state/proposal.rs

```diff
@@ -102,6 +106,19 @@ pub enum VoteType {
     },
 }
 
+/// Type of MultiChoice.
+#[derive(Clone, Debug, PartialEq, Eq, BorshDeserialize, BorshSerialize, BorshSchema)]
+pub enum MultiChoiceType {
+    /// Normal multi choice type that requires decision for only one option when voting.
```

Comment:
It's not for just one option when voting. A user can approve multiple options and as a result of voting on an option the user's full weight is allocated to each approved option

Reply:
Thanks. I changed the explanation comment and added a simple cargo test.